Source/Particles/Collision/BinaryCollision/BinaryCollisionUtils.cpp

```diff
@@ -108,11 +108,27 @@ namespace BinaryCollisionUtils{
             amrex::ParmParse pp_collision_name(collision_name);
             std::string type;
             pp_collision_name.get("type", type);
+            amrex::Vector<std::string> product_species_name;
+            pp_collision_name.getarr("product_species", product_species_name);
+
             if (type == "nuclearfusion") {
                 NuclearFusionType fusion_type = get_nuclear_fusion_type(collision_name, mypc);
                 return nuclear_fusion_type_to_collision_type(fusion_type);
             }
-            amrex::Abort(type + " is not a valid type of collision that creates new particles");
+            if (type == "photonphoton") {
+                WARPX_ALWAYS_ASSERT_WITH_MESSAGE(
+                    product_species_name.size() == 2u,
+                    "ERROR: Photon-photon collisions must contain exactly two product species");
+                auto& product_species1 = mypc->GetParticleContainerFromName(product_species_name[0]);
+                auto& product_species2 = mypc->GetParticleContainerFromName(product_species_name[1]);
+                WARPX_ALWAYS_ASSERT_WITH_MESSAGE(
+                    (product_species1.AmIA<PhysicalSpecies::electron>() && product_species2.AmIA<PhysicalSpecies::positron>())
+                    ||
+                    (product_species1.AmIA<PhysicalSpecies::positron>() && product_species2.AmIA<PhysicalSpecies::electron>()),
+                    "ERROR: Product species of photon-photon collisions must be of type electron and positron");
+                return CollisionType::PhotonPhotonToElectronPositron;
+            }
+            amrex::Abort(type + " is not a valid type of collision that create new particles");
```

Comment:
```suggestion
            amrex::Abort(Utils::TextMsg::Err(type + " is not a valid type of collision that create new particles"));
```
I would suggest adding this, so that all the error messages have the same formatting. If you accept this suggestion, you'll probably need to include `Utils/TextMsg.H`

Reply:
I've accepted these changes by hand since I couldn't do it from here. 
In any case, we should remember that this issue appears other times in the same file.
I will try to fix with another PR.